modules/ximgproc/samples/thinning.cpp

```diff
@@ -10,7 +10,7 @@ using namespace cv;
 
 int main()
 {
-    Mat img = imread("opencv-logo.png", IMREAD_COLOR);
+    Mat img = imread("../data/opencv-logo.png", IMREAD_REDUCED_COLOR_2);
```

Comment:
hi @StevenPuttemans 
"../data/" used in many sample 
for example https://github.com/opencv/opencv_contrib/blob/master/modules/ximgproc/samples/disparity_filtering.cpp#L18-L20

indeed i don't know another way and i always move the opencv/samples/data folder to a proper place according to where the compiled exe files placed.

so still i see this change will be useful.

also IMREAD_REDUCED_COLOR_2 will load the image resized. it is also better to see result image.